test/integration/minishift.go

```diff
@@ -214,7 +214,7 @@ func (m *Minishift) rolloutServicesSuccessfully(servicesToCheck string) error {
 	}
 
 	if !success {
-		return fmt.Errorf("Not all successfully rolled out")
+		return errors.New("Not all successfully rolled out")
```

Comment:
This should actually be changed to keep records of the returned value to determine which service did NOT successfully deploy.